DBDebugToolkit/Classes/Resources/Files/DBFilesTableViewController.m

```diff
@@ -144,6 +144,18 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
         filesTableViewController.path = [self.path stringByAppendingPathComponent:subdirectoryName];
         filesTableViewController.title = subdirectoryName;
         [self.navigationController pushViewController:filesTableViewController animated:YES];
+    } else {
+        NSString *filePath = [self.path stringByAppendingPathComponent:self.files[indexPath.row]];
+        NSData *fileData = [NSData dataWithContentsOfFile:filePath];
```

Comment:
Here is it better to provide the file URL to the UIActivityViewController, otherwise there not all files are correct handeled.
`NSURL *fileURL = [NSURL fileURLWithPath:filePath];`